### ISSUE TITLE
Marko 4 unrecognized tag thrown for nested tag transformers

### ISSUE THREAD

**Issue by mlrawlings** [eBay/ebayui-core#13]
When using a component that has nested tags that are created using the nested tag transformer (like `<ebay-menu-item>` rather than `<ebay-menu:item>`, Marko 4 will throw an error during the parse stage (which occurs before the transform stage) saying it found an unrecognized tag.

**Solution:**
Add an empty tag definition for all valid nested tags:
```json
{
    "<ebay-menu-item>": {}
}
```

**Comment by seangates** [comment]
Would that be alongside the other tag definition? e.g.:

```
{
    "<ebay-menu>": {
        "renderer": "./index.js",
        "transformer": "../../common/transformer/index.js"
    },
    "<ebay-menu-item>": {}
}
```

**Comment by ianmcburnie** [comment]
Closing as fixed in 0.1.0 branch. Will merge to master branch at release time.
